MediaBrowser.Model/Dlna/StreamBuilder.cs

```diff
@@ -665,251 +582,405 @@ private StreamInfo BuildVideoItem(MediaSourceInfo item, VideoOptions options)
+            int? channels = stream?.Channels;
+            int? sampleRate = stream?.SampleRate;
 
-                var isFirstAppliedCodecProfile = true;
-                foreach (var i in options.Profile.CodecProfiles)
+            foreach (ProfileCondition condition in conditions)
+            {
+                switch (codecType)
                 {
-                    if (i.Type == CodecType.Video && i.ContainsAnyCodec(transcodingProfile.VideoCodec, transcodingProfile.Container))
-                    {
-                        bool applyConditions = true;
-                        foreach (ProfileCondition applyCondition in i.ApplyConditions)
+                    case CodecType.Audio:
                         {
-                            int? width = videoStream?.Width;
-                            int? height = videoStream?.Height;
-                            int? bitDepth = videoStream?.BitDepth;
-                            int? videoBitrate = videoStream?.BitRate;
-                            double? videoLevel = videoStream?.Level;
-                            string videoProfile = videoStream?.Profile;
-                            float videoFramerate = videoStream == null ? 0 : videoStream.AverageFrameRate ?? videoStream.AverageFrameRate ?? 0;
-                            bool? isAnamorphic = videoStream?.IsAnamorphic;
-                            bool? isInterlaced = videoStream?.IsInterlaced;
-                            string videoCodecTag = videoStream?.CodecTag;
-                            bool? isAvc = videoStream?.IsAVC;
-
-                            TransportStreamTimestamp? timestamp = videoStream == null ? TransportStreamTimestamp.None : item.Timestamp;
-                            int? packetLength = videoStream?.PacketLength;
-                            int? refFrames = videoStream?.RefFrames;
-
-                            int? numAudioStreams = item.GetStreamCount(MediaStreamType.Audio);
-                            int? numVideoStreams = item.GetStreamCount(MediaStreamType.Video);
-
-                            if (!ConditionProcessor.IsVideoConditionSatisfied(applyCondition, width, height, bitDepth, videoBitrate, videoProfile, videoLevel, videoFramerate, packetLength, timestamp, isAnamorphic, isInterlaced, refFrames, numVideoStreams, numAudioStreams, videoCodecTag, isAvc))
+                            if (!ConditionProcessor.IsVideoAudioConditionSatisfied(condition, channels, bitrate, sampleRate, bitDepth, profile, isSecondaryAudio))
                             {
-                                // LogConditionFailure(options.Profile, "VideoCodecProfile.ApplyConditions", applyCondition, item);
-                                applyConditions = false;
-                                break;
+                                yield return condition;
                             }
+                            break;
                         }
+                    case CodecType.VideoAudio:
+                        {
```

Comment:
```suggestion
                    case CodecType.VideoAudio:
                    {
```

the indentation for these `case` statements is off,
and "Closing brace should be followed by blank line" as the GH analyzer is reporting

Reply:
If you look at other parts of the file, the `case` braces look exactly like this. This is also how the formatter formats the braces.

I did fix the closing brace blank line warning though.